share/esmf_wrf_timemgr/ESMF_TimeMod.F90

```diff
@@ -1080,16 +1080,20 @@ function ESMF_TimeDiff(time1, time2)
 
       ! Can either be yr/month based diff if diff is only in year and month
       ! or absolute seconds if diff in day/seconds as well
-
-      if (d1 == d2 .and. s1 == s2) then
-!         write(6,*) 'tcx timedifft ym'
-         ESMF_TimeDiff%YR = y1 - y2
-         ESMF_TimeDiff%MM = m1 - m2
-         cmptime%S  = 0
-         cmptime%Sn = 0
-         cmptime%Sd = 0
-         ESMF_TimeDiff%basetime = cmptime
-      else
+      !
+      ! Update: Actually, the timeintcmp() routine in ESMF_TimeIntervalMod.F90 is not capable
+      !    of comparing time intervals when one interval has a different year and month than
+      !    the other. So, it is best here to always compute the interval as day/seconds.
+
+!      if (d1 == d2 .and. s1 == s2) then
```

Comment:
Not sure why git is showing a diff here.  We made no mods to this file since cime5.1.